packages/core/src/components/menu/menu.tsx

```diff
@@ -21,7 +21,7 @@ import { polyfill } from "react-lifecycles-compat";
 import { AbstractPureComponent2, Classes, IRef } from "../../common";
 import { DISPLAYNAME_PREFIX, Props } from "../../common/props";
 import { MenuDivider } from "./menuDivider";
-// this cyclic import can be removed in v4.0 (https://github.com/palantir/blueprint/issues/3829)
+// this cyclic import can be removed in v5.0 (https://github.com/palantir/blueprint/issues/3829)
```

Comment:
same as above, this is actually removed in v4.0, so this line should not change. see palantir/blueprint/wiki/Blueprint-4.0